features/lorawan/LoRaWANStack.h

```diff
@@ -393,10 +393,54 @@ class LoRaWANStack: private mbed::NonCopyable<LoRaWANStack> {
      */
     lorawan_status_t stop_sending(void);
 
+    /** Gives access to network provided GPS time
+     *
+     * Network provides a time-stamp for the device synchronization on demand
+     * using GPS time base. The request may originate from Application layer
+     * Clock Synchronization protocol (for v1.0.2) or stack level DevTimeReq MAC
+     * command (for v.10.3 and above). If the request originated from the application
+     * layer, application is responsible for relaying the information to the stack.
+     * In case of DevTimeReq MAC command, the stack will take care of it automatically.
+     *
+     * The API looks up the stored GPS time and the monotonic tick time-stamp taken
+     * at the moment of storing GPS time, and returns GPS time + difference of stored
+     * tick time and current tick time.
+     * If the GPS time was not set by the network yet, the API returns zero.
+     *
+     * @return Current GPS time in milliseconds
```

Comment:
Seems like the "in milliseconds" shouldn't need to be part of the API, if it was properly specified in the definition of `lorawan_time_t`. But it isn't?

Reply:
This is a wrong comment. Current GPS time is in seconds.